roxctl/common/printer/tabularprinter_factory.go

```diff
@@ -67,10 +68,16 @@ func (t *TabularPrinterFactory) AddFlags(cmd *cobra.Command) {
 	cmd.PersistentFlags().BoolVar(&t.Merge, "merge-output", t.Merge, "Merge duplicate cells in prettified tabular output")
 	cmd.PersistentFlags().StringSliceVar(&t.Headers, "headers", defaultImageScanHeaders, "Headers to print in tabular output"+
 		"Will propagate headers to the table unless --row-jsonpath-expressions is also set.")
+	cmd.PersistentFlags().StringSliceVar(&t.RequiredHeaders, "required-headers", []string{}, "Headers denoted as required"+
+		"must be present in a row of the output table, or that row will not be displayed.")
+	err := t.mapRequiredHeadersToJson()
+	if err != nil {
+		cmd.PrintErrf("%v", err)
```

Comment:
should we have a newline here? How this error looks like? Should we break on error?

Reply:
Good catch, left over from debugging when I wrote this